src/structures/MessageSelectMenu.js

```diff
@@ -94,6 +101,47 @@ class MessageSelectMenu extends BaseMessageComponent {
     return this;
   }
 
+  /**
+   * Sets the type of the select menu
+   * @param {SelectMenuComponentType} type Type of the select menu
+   * @returns {MessageSelectMenu}
+   */
+  setType(type) {
+    if (!SelectMenuComponentTypes[type]) throw new TypeError('INVALID_TYPE', 'type', 'SelectMenuComponentType');
+    this.type = BaseMessageComponent.resolveType(type);
+    return this;
+  }
+
+  /**
+   * Adds the channel type to the select menu
+   * @param {...ChannelType[]} channelType Added channel type
+   * @returns {MessageSelectMenu}
+   */
+  addChannelTypes(...channelTypes) {
+    if (!channelTypes.every(channelType => ChannelTypes[channelType])) {
+      throw new TypeError('INVALID_TYPE', 'channelTypes', 'Array<ChannelType>');
```

Comment:
Don't think `Array<ChannelType>` is the right thing here

Reply:
idk